### ISSUE TITLE
Why can't I reproduce the results from the same seed?

### ISSUE THREAD

**Issue by wuxiaofei01** [stevenlsw/contactgen#3]
![image](https://github.com/stevenlsw/contactgen/assets/61744147/9bbfd31a-c92d-49ee-95fa-342d375422f2)

**The configurations on the left and right are exactly the same, except that they run for 3 minutes before and after**

Dear author,

Hello, I am a first-year graduate student. While attempting to reproduce your work, I encountered an issue where using the **same seed** does not lead to an **exact replication of the previous results**. I am wondering why this might be the case. I would greatly appreciate it if you could provide an explanation.

Thank you in advance for your response.

**Comment by stevenlsw** [comment]
It might because of the shuffling in training data loader.

**Comment by wuxiaofei01** [comment]
Hello author, I tried setting shuffle =False in the Dataloader, but this problem still occurred. May I ask why?

**Comment by stevenlsw** [comment]
Hi, you could refer to https://pytorch.org/docs/stable/notes/randomness.html.
